File: src/main/java/ebs/tools/patcher/IoUtil.java
package ebs.tools.patcher;

import java.io.IOException;
import java.io.InputStream;
import java.io.OutputStream;

public class IoUtil {

    public static void copy(InputStream in, OutputStream out) throws IOException {

        int c;
        while ((c = in.read()) != -1) {
            out.write(c);
        }

        in.close();
    }
}

File: src/main/scala/org/bohdi/tools/patcher/IoUtil.scala
package org.bohdi.tools.patcher

import java.io.InputStream
import java.io.OutputStream

object IoUtil {
  def copy(in: InputStream, out: OutputStream) {
    var c: Int = 0
    while ((({
      c = in.read; c
    })) != -1) {
      out.write(c)
    }
    in.close
  }
}